pkg/services/grafana-apiserver/auth/authorizer/org/org_role.go

```diff
@@ -28,25 +28,25 @@ func (auth OrgRoleAuthorizer) Authorize(ctx context.Context, a authorizer.Attrib
 
 	switch signedInUser.OrgRole {
 	case org.RoleAdmin:
-		return authorizer.DecisionAllow, "", nil
+		return authorizer.DecisionNoOpinion, "", nil
```

Comment:
why wouldn't we say allow here?

Reply:
the problem is that this clobers every other decision -- so being an editor does not mean you can edit everything, it only means you have the option to edit (more FGAC can/will have more explicit options)

Yes the fallback to allow all is odd, but I think the alternative is using something other than `union` as our base.  Perhaps we want one that checks if anything says Deny, then if anything says Allow?  (but that would not work with `NewPrivilegedGroups(user.SystemPrivilegedGroup)`)

Reply:
note the other change is to return Deny -- when the role is unkown

Reply:
> being an editor does not mean you can edit everything, it only means you have the option to edit
ah ok, that makes sense

>  FGAC can/will have more explicit options
👍 so it sounds like we could make that the final decision once it's available?

Reply:
Yes -- where FGAC has rules, it can say yes|no and be the final decision, but when it does not this will still be the fallback path